templates/terraform/environments/prod/main.tf

```diff
@@ -122,4 +122,8 @@ module "prod" {
   ]
 
   user_role_mapping = data.terraform_remote_state.shared.outputs.user_role_mapping
+
+  # Cloud trail configuration
```

Comment:
Maybe a comment here about what cloudtrail does, why you might want it enabled, and the cost?